### PR TITLE
docs: Update getting started

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/get-started-dotnet-new.md [hunk 1]
```diff
@@ -10,36 +10,41 @@ To install the templates, type the following:
...
-A basic example:
+To create a blank template, using minimal dependencies:
+```
+dotnet new dotnet new unoapp -preset=blank -o test
+```
+
+To create a recommended template template, using [Uno.Extensions](xref:Overview.Extensions):
 ```
-dotnet new unoapp -o MyApp
+dotnet new dotnet new unoapp -preset=recommended -o test
```
Comment (lukeblevins) [inline doc/articles/get-started-dotnet-new.md:47]:
```suggestion
dotnet new unoapp -preset=recommended -o test
```

File: doc/articles/get-started-dotnet-new.md [hunk 2]
```diff
@@ -10,36 +10,41 @@ To install the templates, type the following:
...
-This template can be used to create a blank multi-platform application for iOS, Android, WebAssembly, macOS, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
+This template can be used to create a multi-platform application for iOS, Android, WebAssembly, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
 
-This template uses a single project head for iOS, Android, macOS, and Mac Catalyst. It requires Visual Studio 2022.
+If comes with the **Blank** and **Recommended** presets.
 
-A basic example:
+To create a blank template, using minimal dependencies:
+```
+dotnet new dotnet new unoapp -preset=blank -o test
```
Comment (lukeblevins) [inline doc/articles/get-started-dotnet-new.md:42]:
```suggestion
dotnet new unoapp -preset=blank -o test
```

File: doc/articles/get-started-dotnet-new.md [hunk 3]
```diff
@@ -10,36 +10,41 @@ To install the templates, type the following:
...
+```
+
+To create a recommended template template, using [Uno.Extensions](xref:Overview.Extensions):
```
Comment (lukeblevins) [inline doc/articles/get-started-dotnet-new.md:45]:
```suggestion
To create from a recommended template, using [Uno.Extensions](xref:Overview.Extensions):
```

File: doc/articles/get-started-dotnet-new.md [hunk 4]
```diff
@@ -10,36 +10,41 @@ To install the templates, type the following:
...
 [!include[getting-help](use-uno-check-inline.md)]
 
-## Uno Platform Blank Application for WinAppSDK - WinUI 3
+## Uno Platform Application
 
-This template can be used to create a blank multi-platform application for iOS, Android, WebAssembly, macOS, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
+This template can be used to create a multi-platform application for iOS, Android, WebAssembly, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
 
-This template uses a single project head for iOS, Android, macOS, and Mac Catalyst. It requires Visual Studio 2022.
+If comes with the **Blank** and **Recommended** presets.
```
Comment (lukeblevins) [inline doc/articles/get-started-dotnet-new.md:38]:
```suggestion
It comes with the **Blank** and **Recommended** presets.
```

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/4.8

**Pull request merged**